iron-iconset-svg.html

```diff
@@ -136,6 +136,10 @@
      */
     _nameChanged: function() {
       new Polymer.IronMeta({type: 'iconset', key: this.name, value: this});
+      this.async(function() {
```

Comment:
@rictic @samccone If we fire synchronously in `_nameChanged`, we are technically not guaranteed that the element has full initialized, because `_nameChanged` can be called synchronously as the initial values of properties are assigned during construction. The potential error condition this creates looks like this:

1) Iconset instance is created
2) Mid-creation, the initial value of `name` is assigned (however, `size` is not yet assigned)
3) Change callback is called, so `iron-iconset-added` is fired
4) Listener responding to `iron-iconset-added` attempts to measure the size of the iconset. This value will be incorrect at the time that the event is handled because it has not yet been initialized.

There are a number of ways to guard against this, and I am inclined to agree that using `async` in this case is an easy way to do it because it guarantees that all initialization has completed by the time the event fires.